file_paths.md

```diff
@@ -208,111 +276,86 @@ create a `puppet` user or group.
-
-    C:\Program Files\Puppet Labs\Puppet\hiera *
-        bin *
-            hiera *                           # ruby bin wrapper
-        lib *
-            hiera.rb *
-
-    C:\Program Files\Puppet Labs\Puppet\mcollective *
-        bin *
-            mcollectived *                    # ruby bin wrapper
-            mco *
-        lib *
-            mcollective.rb *
-
-    C:\Program Files\Puppet Labs\Puppet\misc *
-        LICENSE.rtf *                         # license
-        puppetlabs.ico *                      # icon for start menu shortcut
-        puppetres.dll *                       # event log message resource dll
-        versions.txt *                        # versions of components
-
-    C:\Program Files\Puppet Labs\Puppet\puppet *
-        bin *
-            puppet *                          # ruby bin wrapper
-        lib *
-            puppet.rb *
 
-    C:\Program Files\Puppet Labs\Puppet\pxp-agent *
-        bin *
-            pxp-agent.exe *
-            lib*.dll *                        # gcc, mingw, boost libraries
-                                              # pxp-agent also loads openssl libraries (below)
+        include
+            boost
+            cpp-pcp-client
+            curl
+            facter *                          # facter headers
+            leatherman
+            openssl
+            ruby-2.1.0
+
+        lib
+            cmake
+            engines
+            leatherman*.a
+            libcpp-pcp-client.so.a
+            libcrypto.a
+            libcrypto.dll.a
+            libcurl.dll.a
+            libcurl.la
+            libfacter.so.a
+            libpxp-agent.a
+            libssl.a
+            libssl.dll.a
+            libx64-msvcrt-ruby210.dll.a
+            libx64-msvcrt-ruby210-static.a
+            pkgconfig
+                libcrypto.pc
+                libcurl.pc
+                libssl.pc
+                openssl.pc
+                ruby-2.1.pc
+            ruby
+                2.1.0
+                gems
+                    2.1.0
+                        specifications
+                            facter.gemspec
+                            hiera.gemspec
+                            puppet.gemspec
+                vendor_ruby
+                    facter.rb *               # facter ruby bindings
+                    hiera.rb
+                    hiera/*
+                    hiera_puppet.rb
+                    puppet.rb
+                    puppet/*
+                    mcollective.rb
+                    mcollective/*
+                    puppet.rb
+                    puppet/*
+                    puppet_x.rb
+                    semver.rb
         modules *
-            pxp-module-puppet *
-            pxp-module-puppet.bat *
-
-    C:\Program Files\Puppet Labs\Puppet\service *
-        daemon.rb *                           # windows service daemon
-        nssm.exe *                            # NSSM used to run pxp-agent
```

Comment:
This will also need to be installed, so we'll need to add an nssm.rb file to puppet-agent